project/profiles/capacity4more/modules/c4m/features/c4m_features_footer/includes/c4m_features_footer.theme.inc

```diff
@@ -58,6 +65,38 @@ function theme_c4m_footer_bar(array $variables) {
 }
 
 /**
+ * Theme function OG Project pages footer - uses project footer template.
+ *
+ * @return html
+ *   Rendered html.
+ */
+function theme_c4m_footer_bar_project() {
+  $options = array(
+    'alias' => TRUE,
+    'html' => TRUE,
+    'attributes' => array()
+  );
+  $europaid_link = l(t('International cooperation and development') . ' >', 'https://ec.europa.eu/europeaid/home_en');
+
+  $options['attributes']['class'] = array('text-orange');
+  $project_link = l(t('Capacity4dev') . ' >',  base_path() . 'projects', $options);
+
+  $options['attributes']['title'] = 'Home';
+  $options['attributes']['class'] = array('logo');
+  $logo_img = '<img src="' . file_create_url(drupal_get_path('theme', 'kapablo') . '/project_logo.png') . '" alt="' . t('Home') . '"/>';
```

Comment:
We probably want to use 
https://api.drupal.org/api/drupal/includes!theme.inc/function/theme_image/7.x